.github/workflows/release.yaml

```diff
@@ -0,0 +1,71 @@
+# Prefect Operator release workflow.
+name: Release
+
+on:
+  release:
+    types: [published]
```

Comment:
i'm fine either way, but i think we should be consistent with our lists. either brackets or with dashes as done below

Reply:
Maybe we should throw the `yamllint` pre-commit on this repo?  There's a ton of YAML in here

Reply:
Good shout, updated in 519e4af

Reply:
Added linting setup in 0ccd597875298df.